tests/test_utils/test_transactiontestcase.py

```diff
@@ -44,3 +82,37 @@ def test_queries_cleared(self):
         """
         for alias in connections:
             self.assertEqual(len(connections[alias].queries_log), 0, 'Failed for alias %s' % alias)
+
+
+class TestDataRestoredOnTearDownIfSerializedRollback(TestSerializedContentMockMixin, TransactionTestCase):
+    """
+    Initial data should be recreated in TransactionTestCase._fixture_teardown()
+    after the database is flushed so it's available in next test.
+    """
+    available_apps = ['test_utils']
+    _next_serialized_rollback = True
+    initial_data_migration = '[{"model": "test_utils.car", "pk": 666, "fields": {"name": "K 2000"}}]'
+
+    def _post_teardown(self):
+        super(TestDataRestoredOnTearDownIfSerializedRollback, self)._post_teardown()
+        self.assertTrue(Car.objects.exists())
+
+    def test_that_should_be_the_only_one_in_this_test_case(self):
+        pass
```

Comment:
Why the empty test method?

Reply:
Without test, the `_post_teardown` won't be run.

Reply:
Perhaps worth adding a comment to that effect?

Also, is there a reason for that particular name for the test method? Does it *have* to be the only one?